client/my-sites/upgrades/checkout-thank-you/domain-registration-details.jsx

```diff
@@ -16,7 +16,7 @@ import supportUrls from 'lib/url/support';
 const DomainRegistrationDetails = ( { selectedSite, domain, purchases } ) => {
 	const googleAppsWasPurchased = purchases.some( isGoogleApps ),
 		domainContactEmailVerified = purchases.some( purchase => purchase.isEmailVerified ),
-		hasOtherPrimaryDomain = selectedSite.options.is_mapped_domain && selectedSite.domain !== domain;
+		hasOtherPrimaryDomain = selectedSite.options && selectedSite.options.is_mapped_domain && selectedSite.domain !== domain;
```

Comment:
Well @Tug found that [clearing the local storage](https://github.com/Automattic/wp-calypso/pull/3844#discussion_r55976978) would make `selectedSite.options` undefined. Isn't it a legitimate case?